services/src/functional-tests/src/test/groovy/org/openkilda/functionaltests/spec/switches/MetersSpec.groovy

```diff
@@ -358,6 +359,27 @@ class MetersSpec extends BaseSpecification {
         ]
     }
 
+    @Ignore("https://github.com/telstra/open-kilda/issues/2043")
+    def "Try to reset meters for unmetered flow"() {
+        given: "A flow with the 'bandwidth: 0' and 'ignoreBandwidth: true' fields"
+        def availableSwitches = topology.activeSwitches
+        def src = availableSwitches[0]
+        def dst = availableSwitches[1]
+        assumeTrue("Desired switch combination is not available in current topology", src && dst)
+
+        def flow = flowHelper.randomFlow(src, dst)
+        flow.ignoreBandwidth = true
+        flow.maximumBandwidth = 0
+        flowHelper.addFlow(flow)
+
+        when: "Resetting meter burst and rate to default"
+        northbound.resetMeters(flow.id)
+//        TODO add then and check response: body or message
```

Comment:
then: "Meters are really reset to defaults"
//TODO(adovhan):  Check response and meters are really reset

Reply:
possibly would be better to use github username?
TODO(adovhan) -> TODO(andriidovhan)

Reply:
It is up to you.

Reply:
Done.